### PR TITLE
Feature/58  update keybinding names

### PR DESCRIPTION
**Pull request by Dabble1SC**

### CODE REVIEW

File: DCS-SR-Client/UI/ClientWindow/InputBindingControl.xaml.cs [hunk 1]
```diff
@@ -35,7 +35,7 @@ public InputDeviceManager InputDeviceManager
         public void LoadInputSettings()
         {
             DeviceLabel.Content = InputName;
-            ModifierLabel.Content = InputName + " Modifier";
```
Comment (gtodd876) [inline DCS-SR-Client/UI/ClientWindow/InputBindingControl.xaml.cs:38]: not sure i follow what is going on in this line

Comment (Dabble1SC) [reply DCS-SR-Client/UI/ClientWindow/InputBindingControl.xaml.cs:38]: image

I removed the input name being included before the name modifier and put in a lot of spaces to "tab" modifier over

Comment (Dabble1SC) [reply DCS-SR-Client/UI/ClientWindow/InputBindingControl.xaml.cs:38]: Originally it said

__________________________
Radio 2
Radio 2 Modifier
---------------------------

Comment (Dabble1SC) [reply DCS-SR-Client/UI/ClientWindow/InputBindingControl.xaml.cs:38]: @gtodd876

Comment (gtodd876) [reply DCS-SR-Client/UI/ClientWindow/InputBindingControl.xaml.cs:38]: is spaces the best way to achieve layout in this situation?

Comment (Dabble1SC) [reply DCS-SR-Client/UI/ClientWindow/InputBindingControl.xaml.cs:38]: It was cheap and very easy to accomplish. Additionally, it's very easy to revert back to something else without needing to change the main XAML formatting. We can always move the different cells around in XAML, but this was pretty easy and involved merely seconds to create.

Comment (gtodd876) [reply DCS-SR-Client/UI/ClientWindow/InputBindingControl.xaml.cs:38]: sounds good

**Pull request merged**